libs/indibase/indiccd.cpp

```diff
@@ -1811,6 +1809,12 @@ void CCD::addFITSKeywords(fitsfile * fptr, CCDChip * targetChip)
         fits_update_key_dbl(fptr, "ROTATANG", RotatorAngle, 3, "Rotator angle in degrees", &status);
     }
 
+    // JJ ed 2020-03-28
```

Comment:
Proper comment would be better here I think.

Reply:
Added